src/Lib/Directory.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
Sure
On Sep 25, 2015 14:31, "Noam Lewis" <notifications@github.com> wrote:

> In src/Lib/Directory.hs
> <https://github.com/ElastiLotem/buildsome/pull/52#discussion_r40420758>:
>
> > @@ -1,3 +1,5 @@
> > +{-# LANGUAGE BangPatterns #-}
> > +{-# LANGUAGE OverloadedStrings #-}
>
> Yea, wanna just change it? I won't be at a computer anytime soon
> … <#15004458a3304954_>
> On Sep 25, 2015 2:27 PM, "Eyal Lotem" <notifications@github.com> wrote:
> In src/Lib/Directory.hs <
> https://github.com/ElastiLotem/buildsome/pull/52#discussion_r40420535>: >
> @@ -1,3 +1,5 @@ > +{-# LANGUAGE BangPatterns #-} > +{-# LANGUAGE
> OverloadedStrings #-} Or mempty, would be nicer to avoid language extension
> just for that? — Reply to this email directly or view it on GitHub <
> https://github.com/ElastiLotem/buildsome/pull/52/files#r40420535>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ElastiLotem/buildsome/pull/52/files#r40420758>.
>